expirationd.lua

```diff
@@ -1311,6 +1311,8 @@ return {
     get_tasks      = get_tasks_obsolete,
     run_task       = run_task_obsolete,
     show_task_list = show_task_list_obsolete,
+
+    _VERSION = require('expirationd-version'),
```

Comment:
A missed documentation entry. You could try to generate the documentation with:
```bash
$ make apidoc
```
and ensure that it has an entry about `_VERSION` constant.

Reply:
Done, now there is a separate section in the docs with this field and description